settings/eslint/eslintrc.js

```diff
@@ -0,0 +1,66 @@
+const commonParserOptions = {
+  ecmaFeatures: {
+    jsx: true,
+  },
+  ecmaVersion: '2020',
+  sourceType: 'module',
+};
+
+const commonRules = {
+  'prettier/prettier': [
+    'error',
+    {
+      singleQuote: true,
+      jsxSingleQuote: true,
+      trailingComma: 'all',
+    },
+  ],
+};
+
+module.exports = {
+  globals: {
```

Comment:
Instead of globals, does this work?

```js
env: {
  browser: true, // Since we run in browser
  jest: true, // Since we wish we had unit tests
}
```

Reply:
Seems to work. Thanks.